libraries/common/cs/update-all-blocks.js

```diff
@@ -0,0 +1,34 @@
+export function updateAllBlocks(
+  blockly,
+  workspace,
```

Comment:
We should make this api get the workspace by itself instead of having it passed in because it should always return the workspace from `Tab.Traps.getWorkspace()` since otherwise, problems such as https://github.com/ScratchAddons/ScratchAddons/pull/7587 may occur. The same might as well be done for blockly as well?

Reply:
I'll change it to only take `tab` as an argument.